### ISSUE TITLE
Removes miner fees on claymore ethereum miner.Do you have the Windows version?

### ISSUE THREAD

**Issue by xiekang0817** [gkovacs/remove_miner_fees#2]
Removes miner fees on claymore ethereum miner.Do you have the Windows version?
Can you hack the Windows version and redirect the developer's wallet address?.
I need your help. Please reply to my message. Thank you。

**Comment by xartin** [comment]
i dont believe this will work natively on windows due to the dependency on linux iptables firewall and networking however i did manage to get this working by running ubuntu server in vmware workstation pro.

**Comment by gkovacs** [comment]
yes correct it won't work with windows, you would need to rewrite it to use something other than nfqueue+iptables because those are linux-specific

**Comment by drdada** [comment]
You can maybe use this: https://github.com/JuicyPasta/Claymore-No-Fee-Proxy
Linux & Windows compatible with python.

**Comment by snofte** [comment]
My claymore amd gold miner yesterday behaved strange.  Program had made an exit during night.
At restart be me,  it started mining to another address !!  Param zwal had been changed (not by me !)
and address was blank.  It has also switched from eu.gold to us.gold.  The new address seems to belong to one of the heavy miners in the pool.  Whats going on ???
